src/hints/mod.rs

```diff
@@ -173,27 +275,47 @@ pub fn starknet_os_input(
 ///
 /// vm_load_program(compiled_class.program, ids.compiled_class.bytecode_ptr)
```

Comment:
With indoc! macro you can:
```
let CHECK_DEPRECATED_CLASS_HASH = indoc! {"
        from starkware.python.utils import from_bytes

        computed_hash = ids.compiled_class_fact.hash
        expected_hash = compiled_class_hash
        assert computed_hash == expected_hash, (
            "Computed compiled_class_hash is inconsistent with the hash in the os_input. "
            f"Computed hash = {computed_hash}, Expected hash = {expected_hash}.")

        vm_load_program(compiled_class.program, ids.compiled_class.bytecode_ptr)"
}
```
Hint keys would be readable, there would be no need to repeat hint code in the comments. I would also keep hint key declaration close to hint implementation, less mental context switching.

Reply:
done